lib/shopify_app/utils.rb

```diff
@@ -20,5 +20,17 @@ def self.fetch_known_api_versions
     rescue ActiveResource::ConnectionError
       logger.error("[ShopifyAPI::ApiVersion] Unable to fetch api_versions from Shopify")
     end
+
+    def self.shop_login_url(shop:, return_to:)
```

Comment:
Do we really not already have a helper for this? Seems unlikely to me that we need to write a new helper for this.

Reply:
Thought this was odd too. But since we're not relying on `LoginProtection` inside of the `HomeController`, we can't leverage those helper functions for constructing the login url.